src/cascadia/TerminalCore/TerminalSelection.cpp

```diff
@@ -310,6 +328,20 @@ Terminal::UpdateSelectionParams Terminal::ConvertKeyEventToUpdateSelectionParams
     return std::nullopt;
 }
 
+bool Terminal::MovingEnd() const noexcept
+{
+    // true --> we're moving end endpoint ("lower")
+    // false --> we're moving start endpoint ("higher")
+    return _selection->start == _selection->pivot;
+}
+
+bool Terminal::MovingCursor() const noexcept
+{
+    // true --> we're moving end endpoint ("lower")
+    // false --> we're moving start endpoint ("higher")
+    return _selection->start == _selection->pivot && _selection->pivot == _selection->end;
+}

```

Comment:
both of the inline comments here say the same thing, but the methods clearly return different things.

Are there better names for this?

Reply:
This was my bad. I guess I just copied the comment but never changed it haha.